docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,171 @@ description: Next.js supports built-in image optimization, as well as third part
   </ul>
 </details>
 
-Since version **10.0.0**, Next.js has a built-in Image Component and Automatic Image Optimization.
+The Next.js Image component, [`next/image`](/docs/api-reference/next/image.md), is an extension of the HTML `<img>` element, evolved for the modern web. It includes a variety of built-in performance optimizations to help you achieve good [Core Web Vitals](https://nextjs.org/learn/seo/web-performance). These scores are an important measurement of user experience on your website, and are [factored into Google's search rankings](https://nextjs.org/learn/seo/web-performance/seo-impact).
 
-The Next.js Image Component, [`next/image`](/docs/api-reference/next/image.md), is an extension of the HTML `<img>` element, evolved for the modern web.
+Some of the optimizations built into the Image component include:
 
-The Automatic Image Optimization allows for resizing, optimizing, and serving images in modern formats like [WebP](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types) when the browser supports it. This avoids shipping large images to devices with a smaller viewport. It also allows Next.js to automatically adopt future image formats and serve them to browsers that support those formats.
+- **Improved Performance:** Always send users correctly sized images, using modern image formats.
+- **Visual Stability:** Prevent [Cumulative Layout Shift](https://nextjs.org/learn/seo/web-performance/cls) automatically.
+- **Faster Page Loads:** Images are only loaded when they enter the viewport, with optional blurry placeholders
+- **Asset Flexibility:** On-demand image resizing, even for images stored on remote servers
 
-Automatic Image Optimization works with any image source. Even if the image is hosted by an external data source, like a CMS, it can still be optimized.
+## Using the Image Component
 
-Instead of optimizing images at build time, Next.js optimizes images on-demand, as users request them. Unlike static site generators and static-only solutions, your build times aren't increased, whether shipping 10 images or 10 million images.
+To add an image to your application, import the [`next/image`](/docs/api-reference/next/image.md) component:
 
-Images are lazy loaded by default. That means your page speed isn't penalized for images outside the viewport. Images load as they are scrolled into viewport.
+```jsx
+import Image from 'next/image'
+```
 
-Images are always rendered in such a way as to avoid [Cumulative Layout Shift](https://web.dev/cls/), a [Core Web Vital](https://web.dev/vitals/) that Google [uses in search ranking](https://developers.google.com/search/blog/2020/05/evaluating-page-experience).
+Now, you can define the `src` for your image (either local or remote).
 
-## Image Component
+### Local Images
 
-To add an image to your application, import the [`next/image`](/docs/api-reference/next/image.md) component:
+To use a local image, `import` your `.jpg`, `.png`, or `.webp` files:
```

Comment:
Can we add a note here mention webpack 5 is required for image imports to work?

Reply:
Done!